Nos/Views/DiscoverView.swift

```diff
@@ -170,22 +168,9 @@ struct DiscoverView: View {
                 }
             }
             .animation(.easeInOut, value: columns)
+            .modifier(DoubleTapToPopModifier(tab: .discover))
```

Comment:
I think the convention is to put this part into a function on an extension of `View`. This makes it more expressive at the call site. (`NosNavigationBarModifier` is a good example). Are you open to making that change?

Reply:
Sure